environment/learnrails/spec/models/articles_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe ArticlesController, type: :model do
+RSpec.describe Article, type: :model do
   describe 'creation' do
     it 'can be created' do
       article = ArticlesController.create(title: "testarticle")
```

Comment:
this should be 
article = Articles.create(title: "testarticle")

can you test scenario when article is created without title and expect that error will be raised

      expect {article.save! }.to raise_error(ActiveRecord::RecordInvalid)